pyisy/isy.py

```diff
@@ -159,6 +159,17 @@ def _auto_reconnecter(self):
 
         self._reconnect_thread = None
 
+    def query(self, node=None):
```

Comment:
For this, the parameter is named node, but it looks like it's actually expecting a node address. Perhaps rename this parameter, or change it so that it accepts a node and reads out the address here?

Reply:
Address is better, that's the intent. Updated.